csrc/scheduler/reduction_heuristic.h

```diff
@@ -103,6 +103,9 @@ class ReductionParams : public HeuristicParams {
   // Unrolling/Vectorization factor for outer reduction factor
   int64_t unroll_factor_outer_reduction = 1;
 
+  // Extra unroll on top of vectorization
+  int64_t unroll_factor_top_of_vectorization = 1;
```

Comment:
nitpick: I think this belongs to be right beneath the declaration of `vectorize_inner_reduction`.

Should this also be renamed to identify that this is associated with `inner_reduction` in the name? in case we decide to expand similar support for `iter_dom`. i.e. `vectorize_iter_dom` and `unroll_factor_iter_dom`?

Reply:
Good point. Moved to after `unroll_factor_inner_reduction (this is the vectorization factor)`.

For name, we don't need to change, The reason is it only applies to inner dim, only inner dim can be vectorized. It doesn't matter this inner dim is reduction or iteration.
For outer dim, we already have `unroll_factor_iter_dom`, and outer dim can't be vectorized.

Reply:
Got'ya.